### PR TITLE
Balance, updating quests, fixing typos, fixing scaling/UI for cthun ability.

### PR DESCRIPTION
**Pull request by vampirika**
Added flavour for yogg quests.
Fixed size scaling for cthun major when cthun ability used on it.
Added supply to many faction main buildings.
Lowered uldum level requirement for tolvir quest.
Changed quest objective for elementals from thousand/stonemaul to zul'farrak.
Reduced elemental casters damage.
Reduced armor on reaper.
Reduced stats on ouro.
Added control transfer to AQ after building in outer AQ.
Removed dialogue pop up from fireland invasion.
Minor text fixes.
Added goldmine in outer AQ and reduced gold values in inner AQ.
Added nicknames

### CODE REVIEW

File: src/WarcraftLegacies.Source/PassiveAbilities/SpellConduction/SpellConductionBuff.cs
Comment (YakaryBovine) [inline src/WarcraftLegacies.Source/PassiveAbilities/SpellConduction/SpellConductionBuff.cs:56]: If you haven't already, I'd recommend updating the tooltip to clearly describe the new behaviour; i.e. that both the victim's AND the Obsidian guy's resistances will apply to the damage.

File: src/WarcraftLegacies.Source/Factions/Ahnqiraj.cs
Comment (YakaryBovine) [inline src/WarcraftLegacies.Source/Factions/Ahnqiraj.cs:45]: Good call, we totally forgot these!

File: src/WarcraftLegacies.Source/Quests/Cthun/QuestFiendThousandFaces.cs
Comment (YakaryBovine) [inline src/WarcraftLegacies.Source/Quests/Cthun/QuestFiendThousandFaces.cs:20]: `RewardDescription` should not end with punctuation!
```suggestion
      $"Soldier and Super Major gain the Vampiric Attack ability";
```

File: src/WarcraftLegacies.Source/Quests/Cthun/QuestTanarisOutpost.cs
Comment (YakaryBovine) [inline src/WarcraftLegacies.Source/Quests/Cthun/QuestTanarisOutpost.cs:24]:
```suggestion
      "The sands of Tanaris call for the Hive, but first I must eliminate the Trolls of Zul'farrak.",
```
Just to keep the first-person vibe consistent.

File: src/WarcraftLegacies.Source/Quests/QuestYoggSaron.cs
Comment (YakaryBovine) [inline src/WarcraftLegacies.Source/Quests/QuestYoggSaron.cs:41]:
```suggestion
    protected override string RewardDescription => "Gain the ability to release Yogg-Saron from his prison; he can be slain to acquire Val'anyr, Hammer of Ancient Kings";
```
`RewardDescription` should indicate exactly what happens when the quest is completed!

File: src/WarcraftLegacies.Source/Quests/Skywall/QuestFirelandInvasion.cs
Comment (YakaryBovine) [inline src/WarcraftLegacies.Source/Quests/Skywall/QuestFirelandInvasion.cs:69]: This is such an easy thing to fix I probably wouldn't even bother commenting it out, but your call.

File: src/WarcraftLegacies.Source/Quests/Skywall/QuestThroneWind.cs
Comment (YakaryBovine) [inline src/WarcraftLegacies.Source/Quests/Skywall/QuestThroneWind.cs:24]:
```suggestion
      "We still don't have full control of Skywall. If we defeat the trolls of Zul'Farrak, we could use their power to secure dominion over the Throne of the Four Winds.",
```
Because I don't think this quest is about conquering the _entire_ plane.

**Review by YakaryBovine — DISMISSED** [review]
Looks good, just a few things to fix.
I've approved it so you can decide what you want to fix.

File: src/WarcraftLegacies.Source/Quests/QuestYoggSaron.cs
Comment (vampirika) [reply src/WarcraftLegacies.Source/Quests/QuestYoggSaron.cs:24]: This is the general quest for all players so not needed

File: src/WarcraftLegacies.Source/Quests/Skywall/QuestFirelandInvasion.cs
Comment (vampirika) [reply src/WarcraftLegacies.Source/Quests/Skywall/QuestFirelandInvasion.cs:69]: Intention is to be able to release this and now rush or be blocked by Sebas updating the invasion

File: src/WarcraftLegacies.Source/PassiveAbilities/SpellConduction/SpellConductionBuff.cs
Comment (vampirika) [reply src/WarcraftLegacies.Source/PassiveAbilities/SpellConduction/SpellConductionBuff.cs:56]: Updated

**Review by YakaryBovine — APPROVED** [review]

**Pull request merged**
